admin/main.php

```diff
@@ -0,0 +1,74 @@
+			</svg>
+			<span>Beta Tester</span></a>
+	</div>
+</div>
+<div class="jetpack-beta-container" >
+	<?php Jetpack_Beta_Admin::show_needed_updates(); ?>
+	<?php if ( Jetpack_Beta::get_option() ) {?>
+	<div class="dops-foldable-card is-expanded has-expanded-summary dops-card is-compact">
+		<div class="dops-foldable-card__header has-border">
+			<span class="dops-foldable-card__main">
+				<div class="dops-foldable-card__header-text">
+					<div class="dops-foldable-card__header-text">Currently Running </div>
+				</div>
+			</span>
+			<span class="dops-foldable-card__secondary">
+					<?php Jetpack_Beta_Admin::show_toggle_autoupdates(); ?>
+			</span>
+		</div>
+		<div class="dops-foldable-card__content">
+			<p><?php echo Jetpack_Beta::get_jetpack_plugin_pretty_version(); ?>
+				| <?php echo Jetpack_Beta::get_jetpack_plugin_version(); ?></p>
+		</div>
+	</div>
+	<?php } else {
+		Jetpack_Beta_Admin::start_notice();
+	}
+	?>
+	<div class="dops-foldable-card has-expanded-summary dops-card">
+		<div class="dops-foldable-card__header has-border">
+			<span class="dops-foldable-card__main">
+				<div class="dops-foldable-card__header-text">
+					<div class="dops-foldable-card__header-text"><?php _e( 'Found a bug?', 'jetpack-beta' ); ?></div>
+				</div>
+			</span>
+			<span class="dops-foldable-card__secondary" >
+				<span class="dops-foldable-card__summary">
+					<a type="button" href="<?php echo esc_url( JETPACK_BETA_REPORT_URL ); ?>"
+					   class="is-primary jp-form-button dops-button is-primary is-compact" >
+						<?php _e( 'Report it!', 'jetpack-beta' ); ?>
+					</a>
+				</span>
+			</span>
+		</div>
+	</div>
+	<?php
+	echo '<div class="jetpack-beta__wrap">';
+		Jetpack_Beta_Admin::show_stable_branch();
+	
+		Jetpack_Beta_Admin::show_branch( __( 'Release Candidate' ), 'rc', null, 'rc' );
+		Jetpack_Beta_Admin::show_branch( __( 'Bleeding Edge' ), 'master', null, 'master' );
+		Jetpack_Beta_Admin::show_search_prs();
+		Jetpack_Beta_Admin::show_branches( 'pr' );
+	echo '</div>';
+
+	if ( $to_test = Jetpack_Beta_Admin::to_test_content() ) { ?>
+	<div class="dops-foldable-card is-expanded has-expanded-summary dops-card is-compact">
+		<div class="dops-foldable-card__header has-border">
+				<span class="dops-foldable-card__main">
+					<div class="dops-foldable-card__header-text">
+						<div class="dops-foldable-card__header-text"><?php _e( 'To Test', 'jetpack-beta' ); ?></div>
+					</div>
+				</span>
+		</div>
+		<div class="dops-foldable-card__content">
+			<?php echo $to_test ; ?>
```

Comment:
Since we can render markdown there, it may include some HTML. I don't think we can use `esc_html` here.